.github/workflows/cleanup-pr.yml

```
@@ -0,0 +1,44 @@
+permissions:
+  id-token: write
+  contents: read
+  packages: read
+
+jobs:
+  cleanup:
+    runs-on: ubuntu-latest
+    environment: Development
+    concurrency:
+      group: cleanup-pr-preview-${{ github.event.pull_request.number }}
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)